### PR TITLE
Testimonial  opacity  v2

### PR DESCRIPTION
**Pull request by elixier01**
before | after
-- | --
after | fixed

Opacity is switching on/off when clicking on testimonial faces.

### CODE REVIEW

File: assets/css/index.css [hunk 1]
```diff
@@ -234,16 +234,12 @@ hr {
...
 }
 
-.face__focus__2 {
-opacity: 0.5;
-}
-
-.face__focus__3 {
-opacity: 0.1;
+.faces__active {
+  opacity: 1 !important;
```
Comment (BarryGallagher) [inline assets/css/index.css]: Remove !important if it's not required

**Comment by PhilBroderick** [comment]
Good discussion between @elixier01, @BarryGallagher  and myself over the implementation of this feature. Think it looks good so far! @elixier01 the enhancements we talked about with adding a variable to store the previously clicked testimonial could make it slightly better in that you won't have to remove the `faces-active` from each of the testimonials then add it to the clicked one but happy for this to merge for now!

File: assets/css/index.css (hunk 1 above)
Comment (elixier01) [reply assets/css/index.css]: !important was not required - removed.

**Comment by BarryGallagher** [comment]
If you resolve the conflicts @elixier01 we can merge.

**Pull request merged**